src/modules/cron/cron.controller.ts

```diff
@@ -178,4 +183,49 @@ export class CronController {
             throw new Error('Failed to remove users due to an internal server error.');
         }
     }
+
+    @Put('unlock')
+    @ApiOkResponse({
+        description: 'The users were successfully unlocked.',
+        type: Boolean,
+    })
+    @ApiUnauthorizedResponse({ description: 'Not authorized to unlock users.' })
+    @ApiForbiddenResponse({ description: 'Insufficient permissions to unlock users.' })
+    @ApiNotFoundResponse({ description: 'Insufficient permissions to unlock users.' })
+    @ApiInternalServerErrorResponse({ description: 'Internal server error while trying to unlock users.' })
+    public async unlockUsersWithExpiredLocks(@Permissions() permissions: PersonPermissions): Promise<boolean> {
+        try {
+            const userLocks: UserLock[] = await this.userLockRepository.getLocksToUnlock();
+            if (userLocks.length === 0) {
+                return true;
+            }
+
+            const results: PromiseSettledResult<Result<void, DomainError>>[] = await Promise.allSettled(
```

Comment:
I guess we use Promise.allSettled instead of Promise.all here because we still want to unlock another user even though the previous one/ones fail but apparently the endpoint only returns true/false or throws a generic error so we don't really handle the resolved promises individually for example count the number of users that resolved or failed and log them for metrics... But I guess this is something we could add later for all the endpoints here...